test/bigtest/tests/check-in-with-claimed-returned-test.js

```diff
@@ -85,6 +85,7 @@ describe('CheckIn with claimed returned', () => {
 
       it('Payment status Cancelled item returned', () => {
         expect(acc.paymentStatus.name).to.equal(cancelFeeClaimReturned.CANCEL_PAYMENT_STATUS);
+        expect(acc.remaining).to.equal(0);
```

Comment:
Could you please move` expect(acc.remaining).to.equal(0);` to separate test.
```
it('should set remaining to 0', () => {
   expect(acc.remaining).to.equal(0);
});
```

Reply:
Ok, I will make the changes